node-group-project/server/server.js

```diff
@@ -83,6 +83,13 @@ app.get('/api/v1/pettypes', (req, res) => {
   helpers.getAllFromTable("pet_types", res, pool)
 })
 
+app.get('/api/v1/pets/:pet_type', (req, res) => {
+  let queryString = "SELECT * FROM adoptable_pets LEFT JOIN pet_types ON adoptable_pets.type_id = pet_types.id WHERE type = $1";
```

Comment:
Can you explain why you've chosen a left join here? 
I'm having trouble finding a scenario where you'd want pet types that don't have an adoptable pet.

Reply:
That actually makes a lot of sense and I can't remember why I did a left there.  I'll change it.